src/extensions/default/bramble/lib/MouseManagerRemote.js

```diff
@@ -1,6 +1,21 @@
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true, bitwise: true */
 /* global addEventListener, removeEventListener, sessionStorage */
-(function(transport) {
+
+var testSessionStorage = "testSessionStorage";
```

Comment:
All these lines will need to be indented 4-spaces after you restore that function above.